examples/magic.py

```diff
@@ -0,0 +1,128 @@
+import asyncio
```

Comment:
I've been trying to empty our examples/ folder, so I'd like to understand: what is the purpose of this file?

Should this be an e2e test instead? Should we delete it? Or something else?

Reply:
It can either go away or be an e2e test. I was using it to visually verify that magic was functioning properly. It seemed potentially useful for anyone down the line who was further debugging magic related stuff - but I'm happy to remove it as well.

What's your pref?

Reply:
Sounds like a good candidate for a Cypress test